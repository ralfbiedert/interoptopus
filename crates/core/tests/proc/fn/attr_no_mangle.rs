use interoptopus::ffi;

#[ffi]
#[unsafe(no_mangle)]
fn foo(_: u8) {}

fn main() {}
