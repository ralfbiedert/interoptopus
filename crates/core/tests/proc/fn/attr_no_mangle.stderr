error: Functions with explicit #[no_mangle] are not supported, remove the attribute and let #[ffi] handle it.
 --> tests/proc/fn/attr_no_mangle.rs:4:1
  |
4 | #[unsafe(no_mangle)]
  | ^^^^^^^^^^^^^^^^^^^^
